components/prism-abap.js

```diff
@@ -0,0 +1,50 @@
+		lookbehind: true,
+	},
+	/* Numbers can be only integers. Decimal or Hex appear only as strings */
+	'number' : /\b\d+\b/g,
+	/* Operators must always be surounded by whitespace, they cannot be put 
+	adjacent to operands. 
+	*/
+	'operator' : {
+		pattern: /(\s)(\+|-|\*{1,2}|\/|<|>|<=|>=|=|\?=|<>)(?=\s)/g,
+		lookbehind: true,
+	},
+	'string_operator' : {
+		pattern: /(\s)(&{1,2})(?=\s)/g,
+		lookbehind: true,
+		alias: "keyword"
```

Comment:
That is because in the official ABAP Editor this specific kind of operator is highlighted as keyword.